CodingSession/ImageExtractinRTfield.py

```diff
@@ -112,26 +76,31 @@
             #    html_file.close()
 
             #  replace <img> html tags into explanations text
-            new_text = re.sub(r'<img.+' + re.escape(each_url_one) + r'.+?</img>',
+            new_text = re.sub(r'<img.+' + re.escape(each_url) + r'.+?</img>',
                               '(Please refer to the following case note for screenshot - ' +
                               refid + file_type + ')', new_text)
-    if image_url_two:
+    return new_text, MD5Hash_list
+
+
+def getbase64encodedimage(new_text, md5hash_list):
+    image_url = re.findall('data:image/[png|gif]*;base64\S+"', row[3])
+    if image_url:
         i = 0
-        for each_url_two in image_url_two:
+        for each_url in image_url:
             MD5Hash_dict = {}
             i += 1  # file name identifier to differentiate multiple images in one field
             #  Get base64 encode
-            base64_code = each_url_two.replace('data:image/png;base64,', '').replace('data:image/gif;base64,', '') \
+            base64_code = each_url.replace('data:image/png;base64,', '').replace('data:image/gif;base64,', '') \
                 .replace('"', '').encode('utf-8')
             #  Define the file type
-            if 'data:image/png;base64,' in each_url_two:
+            if 'data:image/png;base64,' in each_url:
                 filename = CaseNumber + '_' + fieldname + '_' + str(i) + '.jpg'
-            elif 'data:image/gif;base64,' in each_url_two:
+            elif 'data:image/gif;base64,' in each_url:
                 filename = CaseNumber + '_' + fieldname + '_' + str(i) + '.gif'
             #  Generate the file path
             path = 'D:/App/Carbon/Salesforce/Image Attachments/' + env + '/Land/' + EID + '/'
```

Comment:
You should put 'D:/App/Carbon/Salesforce/Image Attachments/' in a global constants variable.